magda-web-client/src/Components/Dataset/DatasetPage.tsx

```diff
@@ -31,6 +33,11 @@ const DatasetPage: FunctionComponent<PropsType> = props => {
 
     const publisherId = dataset?.publisher?.id ? dataset.publisher.id : null;
 
+    const editButtonTooltipText =
+        "The `Edit` function is only available to datasets created by Magdad.";
```

Comment:
I think it'd make more sense to flip this around to something like "Datasets harvested from outside Magda can't be edited".

Reply:
👍 will update